build/openshift-container-platform-4/component.yaml

```diff
@@ -11994,13 +11994,12 @@ satisfies:
 - control_key: SI-14
```

Comment:
This change should be removed from your PR.

Reply:
All build/* have been removed from the PR